sklearn/utils/tests/test_cython_templating.py

```diff
@@ -11,10 +11,12 @@ def test_files_generated_by_templates_are_git_ignored():
 
     base_dir = pathlib.Path(sklearn.__file__).parent
     ignored_files = open(gitignore_file, "r").readlines()
-    ignored_files = list(map(lambda line: line.strip("\n"), ignored_files))
+    ignored_files = list(
+        map(lambda line: pathlib.Path(line.strip("\n")), ignored_files)
+    )
```

Comment:
Since we are now comparing paths, may we include the following update to fully use the `pathlib` api?

```python
    ignored_files = gitignore_file.read_text().split("\n")
    ignored_files = [pathlib.Path(line) for line in ignored_files]
```

Reply:
Great finding, thanks, applied!